modules/apps/captcha/captcha-rest-impl/src/main/java/com/liferay/captcha/rest/internal/resource/v1_0/CaptchaResourceImpl.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.captcha.rest.internal.resource.v1_0;
+
+import com.liferay.captcha.configuration.CaptchaConfiguration;
+import com.liferay.captcha.recaptcha.ReCaptchaImpl;
+import com.liferay.captcha.rest.dto.v1_0.FormCaptcha;
+import com.liferay.captcha.rest.internal.util.CaptchaTokenUtil;
+import com.liferay.captcha.rest.resource.v1_0.CaptchaResource;
+import com.liferay.captcha.util.CaptchaUtil;
+import com.liferay.portal.kernel.captcha.Captcha;
+import com.liferay.portal.kernel.captcha.CaptchaTextException;
+import com.liferay.portal.kernel.encryptor.EncryptorUtil;
+import com.liferay.portal.kernel.model.Company;
+import com.liferay.portal.kernel.module.configuration.ConfigurationProvider;
+import com.liferay.portal.kernel.util.Base64;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.io.ByteArrayOutputStream;
+
+import javax.servlet.http.HttpServletRequest;
+
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.NotAcceptableException;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ServiceScope;
+
+/**
+ * @author Loc Pham
+ */
+@Component(
+	properties = "OSGI-INF/liferay/rest/v1_0/captcha.properties",
+	scope = ServiceScope.PROTOTYPE, service = CaptchaResource.class
+)
+public class CaptchaResourceImpl extends BaseCaptchaResourceImpl {
+
+	@Override
+	public com.liferay.captcha.rest.dto.v1_0.Captcha getSimpleCaptcha()
+		throws Exception {
+
+		_checkSimpleCaptchaConfiguration();
+
+		Captcha captcha = CaptchaUtil.getCaptcha();
+
+		ByteArrayOutputStream imageByteArrayOutputStream =
+			new ByteArrayOutputStream();
+
+		String captchaAnswer = captcha.serveImageOutputStream(
+			_httpServletRequest, imageByteArrayOutputStream);
+
+		String base64CaptchaImage =
+			"data:image/png;base64," +
+				Base64.encode(imageByteArrayOutputStream.toByteArray());
+
+		imageByteArrayOutputStream.close();
+
+		return new com.liferay.captcha.rest.dto.v1_0.Captcha() {
```

Comment:
com.liferay.captcha.rest.dto.v1_0.Captcha captcha = new com.liferay.captcha.rest.dto.v1_0.Captcha();
captcha.setCaptchaToken(CaptchaTokenUtil.generateCaptchaToken(_contextCompany, captchaAnswer));
captcha.setImage(base64CaptchaImage);
return captcha;

I think we should change to this common syntax instead of anonymous inner class initializer